mod/manifest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+    use Friendica\Core\Config;
+
+    function manifest_content(App $a) {
+
+		$tpl = get_markup_template('manifest.tpl');
+
+		header('Content-type: application/manifest+json');
+
+		$touch_icon = Config::get('system', 'touch_icon', 'images/friendica-128.png');
+		if ($touch_icon == '') {
```

Comment:
Are you sure? If the touch_icon setting was previously defined but is now empty, Config::get seems to return an empty string and not the default.